development/nautobot_config.py

```diff
@@ -93,5 +93,6 @@
             "rtr": "router",
             "spine": "spine",
         },
+        "platform_name": "Arista EOS",
```

Comment:
this is different from the previous `"arista.eos.eos"`. is it correct?-

Reply:
Yes it should be changed back to what it was before to be backwards-compatible - good point.

Reply:
Actually this raises an interesting question. Before it was
```
name="arista.eos.eos",
slug="arista_eos",
```

Now it would be 
```
name="Arista EOS",
slug="arista_eos"
```

I see two ways of going about this:

1. Changing the default platform name but retaining the slug (also bringing the slug back in line with the slugified version of the name, this is what its like right now)
2. Next to the name, also make the slug of the platform configurable and configure these to be what they were before.

Personally I would prefer option one. What do you think?

Reply:
Actually come to think of it because `name` is in `defaults` for the `get_or_create` call, the first option is even backwards-compatible. I think we should stay with this approach.